Vercel build & deploy error

Env vars are already set. Below is build and deploy error logs from Vercel:
04:12:56.822 Running build in Washington, D.C., USA (East) – iad1
04:12:56.822 Build machine configuration: 4 cores, 8 GB
04:12:56.831 Cloning github.com/cameronking4/acme-corp (Branch: main, Commit: 3863aa1)
04:12:56.832 Skipping build cache, deployment was triggered without cache.
04:12:57.020 Cloning completed: 189.000ms
04:12:57.365 Running "vercel build"
04:12:57.759 Vercel CLI 48.9.0
04:12:58.171 Installing dependencies...
04:13:10.788 npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
04:13:10.837 npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
04:13:11.014 npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
04:13:11.829 npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
04:13:11.931 npm warn deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead
04:13:13.468 npm warn deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.
04:13:18.128 
04:13:18.128 added 411 packages in 20s
04:13:18.128 
04:13:18.128 157 packages are looking for funding
04:13:18.128   run `npm fund` for details
04:13:18.184 Detected Next.js version: 15.0.3
04:13:18.189 Running "npm run build"
04:13:18.298 
04:13:18.298 > acme-corp@0.1.0 build
04:13:18.298 > next build
04:13:18.298 
04:13:18.865 Attention: Next.js now collects completely anonymous telemetry regarding usage.
04:13:18.866 This information is used to shape Next.js' roadmap and prioritize features.
04:13:18.866 You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
04:13:18.867 https://nextjs.org/telemetry
04:13:18.867 
04:13:18.963    ▲ Next.js 15.0.3
04:13:18.963 
04:13:18.984    Creating an optimized production build ...
04:13:23.688 Failed to compile.
04:13:23.688 
04:13:23.688 app/layout.tsx
04:13:23.689 An error occurred in `next/font`.
04:13:23.689 
04:13:23.689 Error: Cannot find module 'autoprefixer'
04:13:23.689 Require stack:
04:13:23.689 - /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js
04:13:23.689 - /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/index.js
04:13:23.689 - /vercel/path0/node_modules/next/dist/build/webpack/config/index.js
04:13:23.689 - /vercel/path0/node_modules/next/dist/build/webpack-config.js
04:13:23.689 - /vercel/path0/node_modules/next/dist/build/webpack-build/impl.js
04:13:23.689 - /vercel/path0/node_modules/next/dist/compiled/jest-worker/processChild.js
04:13:23.689     at Function.<anonymous> (node:internal/modules/cjs/loader:1383:15)
04:13:23.689     at /vercel/path0/node_modules/next/dist/server/require-hook.js:55:36
04:13:23.690     at Function.resolve (node:internal/modules/helpers:157:19)
04:13:23.690     at loadPlugin (/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:49:32)
04:13:23.690     at /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:157:56
04:13:23.690     at Array.map (<anonymous>)
04:13:23.690     at getPostCssPlugins (/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:157:47)
04:13:23.690     at async /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/index.js:124:36
04:13:23.690     at async /vercel/path0/node_modules/next/dist/build/webpack/loaders/next-font-loader/index.js:86:33
04:13:23.690     at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:153:20)
04:13:23.690 
04:13:23.691

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
     "@types/node": "^22",
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "autoprefixer": "^10.4.20",
     "eslint": "^8",
     "eslint-config-next": "15.0.3",
     "postcss": "^8",

